netsim/modules/evpn.py

```diff
@@ -407,7 +397,7 @@ def module_pre_transform(self, topology: Box) -> None:
       module='evpn')
 
   def node_pre_transform(self, node: Box, topology: Box) -> None:
-    validate_no_node_vrf_bundle(node,topology)
+    validate_no_node_vrf_attributes(node,topology)
```

Comment:
Did group data processing happen already before pre_transform? Any reason why we cannot move this to node_post_transform? We're ignoring any transit VNIs specified there anyway, the warning is merely a courtesy to the user no?

Reply:
The per-node EVPN data makes no sense -- bundles mode and transit VNI should be in sync across all nodes in the same VRF.

This check prevents per-node evpn attributes (and resulting troubleshooting) and has to be done before the global VRF values are merged with the node VRF values.